addons/website_sale/views/templates.xml

```diff
@@ -2347,11 +2347,13 @@
                 <div class="oe_structure" id="oe_structure_website_sale_confirmation_3"/>
             </t>
 
-            <div class="d-flex justify-content-between align-items-center">
-                <h3>Thank you for your order.</h3>
-                <a role="button" class="d-none d-md-inline-block btn btn-primary ms-auto" href="/shop/print" target="_blank" aria-label="Print" title="Print"><i class="fa fa-print me-2"></i>Print</a>
-            </div>
             <t t-set="tx_sudo" t-value="order.get_portal_last_transaction()"/>
+            <t t-if="tx_sudo.state in ['pending', 'done']">
+                <div class="d-flex justify-content-between align-items-center">
```

Comment:
```suggestion
            <div t-if="tx_sudo.state in ['pending', 'done']" class="d-flex justify-content-between align-items-center">
```

Reply:
Hello @AntoineVDV 
Would this suggestion solve the "breaking customizations that use one of the removed nodes as a hook" problem"?

Reply:
No, it's just the correct way to work with `t-if/t-else`s: you should only use a `<t>` node if necessary.

Adding nodes can occasionally break customizations (e.g., an XPath targeting the parent's Nth child), and removing nodes has a higher chance of doing it (e.g., an XPath hooking itself on the removed node).

In the end, there is always a non-zero chance of breaking something when we merge a bugfix, but we must mitigate it as much as possible. Here, I was wondering if we could find a less "invasive" fix because the chances that this message has been customized are high.